addons/account/models/account_move.py

```diff
@@ -1308,6 +1310,7 @@ def _compute_amount(self):
             move.amount_tax_signed = -total_tax
             move.amount_total_signed = abs(total) if move.move_type == 'entry' else -total
             move.amount_residual_signed = total_residual
+            move.amount_total_in_currency_signed = abs(move.amount_total) if move.move_type == 'entry' else -(sign * move.amount_total)
```

Comment:
These fields really need a better naming convention, but we can keep this for a later task. (i.e. `amount_total_signed` -> `amount_total_company_signed`)

Reply:
You can't mix stored and non stored fields in the same compute function. Computing the value in the migration script is probably the way to go as it is pretty straightforward.
It might also be useful to filter in the search view, making it necessary to store it.

Reply:
added store=True back